data.py

```diff
@@ -1,3 +1,4 @@
+#! usr/bin/python
```

Comment:
I don't think it will be apropos to the `__init__.py`, but test it in the `__main__.py` maybe? We're not sure what it will do. I expect you to research and get back to me though. :)